docs/guide/conditional-validation.md

```diff
@@ -107,9 +107,9 @@ $rules = [
 $result = (new Validator())->validate($data, $rules);
 ```
 
-If the attribute is required, it's more appropriate to use `skipOnError: true` instead with preceding `Required` rule. 
-This is because empty values' detection within `Required` rule and for skipping in further rules can be set separately 
-(this is described below in more detail, see [Configuring empty criterias in other rules] section).
+If the attribute is required, it is more appropriate to use `skipOnError: true` instead of the preceding `Required` rule.
```

Comment:
`of` is incorrect here.  Explanation:

`skipOnError: true` together with the preceding `Required` rule are used instead of `skipOnEmpty: true`.

Not instead of `Required` rule.